Stripe/STPEphemeralKeyManager.h

```diff
@@ -0,0 +1,46 @@
+//
+//  STPEphemeralKeyManager.h
+//  Stripe
+//
+//  Created by Ben Guo on 5/9/17.
+//  Copyright © 2017 Stripe, Inc. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import "STPEphemeralKeyProvider.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+typedef void (^STPEphemeralKeyCompletionBlock)(STPEphemeralKey * __nullable ephemeralKey, NSError * __nullable error);
+
+@interface STPEphemeralKeyManager : NSObject
+
+/**
+ If the current ephemeral key expires in less than this time interval, a call
```

Comment:
Should we take the time to standardize on a comment format or does it not matter? (some of our stuff has preceding `*` on every line and some don't.

Happy to punt on this to a future PR or release just wanted to mention it as an inconsistency.